public/components/event_analytics/explorer/save_panel/save_panel.tsx

```diff
@@ -50,59 +42,36 @@ export const SavePanel = ({
   savedObjects,
   savePanelName,
   showOptionList,
-  curVisId,
-  spanValue,
   setSubType,
-  metricMeasure,
-  setMetricMeasure,
-  setMetricLabel,
-  metricChecked,
+  isSaveAsMetricEnabled,
 }: ISavedPanelProps) => {
   const [options, setOptions] = useState([]);
   const [checked, setChecked] = useState(false);
-  const [measure, setMeasure] = useState('');
-  const [label, setLabel] = useState([]);
+  const [svpnlError, setSvpnlError] = useState(null);
```

Comment:
What are you doing this this value?

Reply:
the linting rule doesn't allow using console.xxx statement, so here I've just done a quick change to make error as a state (so later we can pipe error to maybe toast UI) rather than using console as this approach may not help on knowing the error.